src/bulkApi/handlers/bulkTransfers.js

```diff
@@ -31,8 +31,9 @@
 const TransferService = require('../../domain/transfer')
 const Logger = require('@mojaloop/central-services-shared').Logger
 const Boom = require('boom')
-const { BulkTransferModel } = require('../models/bulkTransfers/bulkModels')
+const { BulkTransferModel } = require('../../models/mongo/bulkTransfer')
```

Comment:
Why mongo in path?

Reply:
Thanks, will change it to schema as proposed below.

Reply:
Also applied the change to central-ledger PR.